### ISSUE TITLE
Chrome - cross-origin fail to load video picture but plays sound

### ISSUE THREAD

**Issue by ansible-comme** [facebook/react-360#621]
I'm currently hosting my r360 build on my server, but the video is coming from elsewhere.

I've had this build running for around 6 months with no issue, now all of a sudden it's throwing this error in the console and refusing to load video picture (sound still plays):

`THREE.WebGLState: DOMException: Failed to execute 'texImage2D' on 'WebGLRenderingContext': The video element contains cross-origin data, and may not be loaded.
`

The video stream works if I call it from my server as HTML5 `<video>` tag, but not from React360.

I can see there are some similar CORS issues posted, but I'm stumped as to what has changed - this is **only effecting the Chrome browser** currently (Firefox, Oculus browser still run as expected).

* 'react-360' package version: react-360@1.0.0
* 'react-360-web' package version: react-360-web@1.0.2
* Operating System: Windows & MacOS
* Browser: Chrome
* VR Device: browser only

**Comment by larrylin28** [comment]
Please check your server response, is it still returning a correct cross-origin header? 
The "sudden" thing can be related to Chrome version update, if you are able to test on a older version would help identify if it's related.

**Comment by ansible-comme** [comment]
hi @larrylin28  ,

I've tested on older versions of Chrome and can confirm the issue is only with the newest version. All previous versions listed below load and play without problem.

2018-06-07 (67.0.3396.79) ✔
2018-08-01 (68.0.3440.84) ✔
2018-09-16 (69.0.3497.92) ✔
2018-11-6 (70.0.3538.77) ✖

Logged response header:
> Access-Control-Allow-Origin: *

Any thoughts?

**Comment by ansible-comme** [comment]
@larrylin28
I just found this thread on three.js github: https://github.com/mrdoob/three.js/issues/15197

I'm using Kaltura as video host and their API URL generates a redirect to their 'cfvod' URL.
So updating the video src to the redirected URL has solved the issue for me.

Closing this as not a R360 bug. 
Thank you for your help.

**Comment by MisterFOnGitHub** [comment]
I have the same problem.

Loading a video from a KeyCDN server, through threejs, to display on a 3D object, I can hear the sound, see the first image of the video but then I get the error:
THREE.WebGLState: DOMException: Failed to execute 'texImage2D' on 'WebGLRenderingContext': Tainted canvases may not be loaded.
I get this error on chrome Version 86.0.4240.75 (Build officiel) (64 bits), and also on Microsoft Edge. Haven't tested other browsers.

I checked that the cross domain headers where correct. I also have a simple html test page where the video loads properly in a video element, ensuring the cross domain policy is correct.
I also checked that I could load images coming from the same server properly as threejs textures and it works fine.

Has anyone found a solution for this ?
